### ISSUE TITLE
NoClassDefFoundError on start-up

### ISSUE THREAD

**Issue by SlimeDog** [DenizenScript/Depenizen#357]
Spigot or Paper 1.16.5 latest
Citizens 2.0.27-b2275
Denizen 1.1.9-b5444
Depenizen 2.0.0-b690

On server start-up:
```
[21:35:11] [Server thread/INFO]: [Depenizen] Enabling Depenizen v2.0.0 (build 690)
[21:35:11] [Server thread/ERROR]: Error occurred while enabling Depenizen v2.0.0 (build 690) (Is it up to date?)
java.lang.NoClassDefFoundError: com/palmergames/bukkit/towny/exceptions/NotRegisteredException
        at com.denizenscript.depenizen.bukkit.Depenizen.registerCoreBridges(Depenizen.java:128) ~[?:?]
        at com.denizenscript.depenizen.bukkit.Depenizen.onEnable(Depenizen.java:29) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[patched_1.16.5.jar:git-Paper-526]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:380) ~[patched_1.16.5.jar:git-Paper-526]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:483) ~[patched_1.16.5.jar:git-Paper-526]
        at org.bukkit.craftbukkit.v1_16_R3.CraftServer.enablePlugin(CraftServer.java:501) ~[patched_1.16.5.jar:git-Paper-526]
        at org.bukkit.craftbukkit.v1_16_R3.CraftServer.enablePlugins(CraftServer.java:415) ~[patched_1.16.5.jar:git-Paper-526]
        at net.minecraft.server.v1_16_R3.MinecraftServer.loadWorld(MinecraftServer.java:466) ~[patched_1.16.5.jar:git-Paper-526]
        at net.minecraft.server.v1_16_R3.DedicatedServer.init(DedicatedServer.java:241) ~[patched_1.16.5.jar:git-Paper-526]
        at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:941) ~[patched_1.16.5.jar:git-Paper-526]
        at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:175) ~[patched_1.16.5.jar:git-Paper-526]
        at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.ClassNotFoundException: com.palmergames.bukkit.towny.exceptions.NotRegisteredException
        at java.net.URLClassLoader.findClass(URLClassLoader.java:471) ~[?:?]
        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:182) ~[patched_1.16.5.jar:git-Paper-526]
        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:111) ~[patched_1.16.5.jar:git-Paper-526]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:588) ~[?:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:521) ~[?:?]
        ... 12 more
[21:35:11] [Server thread/INFO]: [Depenizen] Disabling Depenizen v2.0.0 (build 690)
```

**Comment by mcmonkey4eva** [comment]
what version of Towny do you have?

**Comment by SlimeDog** [comment]
We don't have Towny.

**Comment by mcmonkey4eva** [comment]
... ah, right, automatic warning fixer in intellij can't read my comment telling it to please not change that.
Woops. Fixed, thanks for the report

**Comment by SlimeDog** [comment]
No worries. Confirmed fixed in Depenizen 2.0.0-b691.
